R/as_phylo.R

```diff
@@ -0,0 +1,42 @@
+#' @rdname as_phylo
+#' @title as_phylo
+#' @description `r lifecycle::badge("deprecated")`
+#'
+#' `as_phylo` methods were renamed `as.phylo`
+#' for more consistency with other R methods.
+#'
+#' @family community
+#' @export
+#' @param x object to be coerced
+#' @inheritParams ape::as.phylo
+as_phylo <- function(x, ...) {
+  lifecycle::deprecate_soft(
+    "1.5.0",
+    "ape::as.phylo()"
+  )
+  UseMethod("ape::as.phylo")
+}
+#' @rdname as_phylo
+#' @method as_phylo communities
+#' @family community
+#' @inheritParams as.phylo.communities
+#' @export
+as_phylo.communities <- function(x, use.modularity = FALSE, ...) {
+  lifecycle::deprecate_soft(
+    "1.5.0",
+    "as.phylo()"
+  )
+  as.phylo.communities(x = x, use.modularity = use.modularity, ...)
+}
+
+#' @rdname as_phylo
+#' @inheritParams ape::as.phylo
+#' @export
+as_phylo.igraphHRG <- function(x, ...) {
+  lifecycle::deprecate_soft(
+    "1.5.0",
+    "as.phylo()"
+  )
+  ape::as.phylo(x, ...)
+}
+rlang::on_load(s3_register("ape::as.phylo", "igraphHRG"))
```

Comment:
Should this live closer to the declaration of the method?

Reply:
You mean `as.phylo.igraphHRG()` ?